src/rgw/rgw_http_client.cc

```diff
@@ -380,7 +435,7 @@ int RGWHTTPClient::wait()
 RGWHTTPClient::~RGWHTTPClient()
 {
   if (req_data) {
-    RGWHTTPManager *http_manager = req_data->get_manager();
+    RGWHTTPManager *http_manager = req_data->mgr;
```

Comment:
@yehudasa I see in `rgw_http_req_data::get_manager()` the mutex::locker is holded, why we no more need it now?